package.json

```diff
@@ -2,18 +2,18 @@
   "name": "edx",
   "version": "0.1.0",
   "dependencies": {
+    "backbone": "~1.3.2",
     "coffee-script": "1.6.1",
     "edx-pattern-library": "~0.12.4",
     "edx-ui-toolkit": "~0.9.1",
-    "requirejs": "~2.1.22",
     "jquery": "~2.2.0",
-    "jquery-migrate": "1.3.0",
+    "jquery-migrate": "^1.4.1",
```

Comment:
Nit: we've been using ``~`` rather than ``^`` so I'd be consistent and switch this over.